### PR TITLE
added static data to myaccount page and footer

### PR DESCRIPTION
**Pull request by vtejaswini01**
Screenshot 2022-01-22 133608

### CODE REVIEW

File: src/app/components/utils/footer/footer.component.html [hunk 1]
```diff
@@ -1 +1,26 @@
-<p>footer works!</p>
+<div class="text-center" id="main">
+    <!--Footer Elements -->
```
Comment (khssupriya) [inline src/app/components/utils/footer/footer.component.html:2]: Comment is not necessary here

File: src/app/components/utils/footer/footer.component.html [hunk 2]
```diff
@@ -1 +1,26 @@
-<p>footer works!</p>
+<div class="text-center" id="main">
```
Comment (khssupriya) [inline src/app/components/utils/footer/footer.component.html:1]: change id from "main" to "footer"

File: src/app/components/utils/footer/footer.component.scss [hunk 3]
```diff
@@ -0,0 +1,12 @@
+@import '../../../../styles.scss';
+#main{
+    background-color:whitesmoke;
+    position: fixed;
+    bottom:0;
+    padding:10px;
+    width:100%;
+}
+
+h5,i{
```
Comment (khssupriya) [inline src/app/components/utils/footer/footer.component.scss:10]: Can we have some padding to all icons?

File: src/app/components/utils/footer/footer.component.html [hunk 4]
```diff
@@ -1 +1,26 @@
...
+    <!--Footer Elements -->
+    <h5 class="text-center">URL Storage</h5> 
+    
+   <div class="container">
+       <ul class="list-unstyled list-inline text-center">
+           <li class="list-inline-item">
+             <a href="#" class="text-dark"><i class="fab fa-facebook"></i></a>
```
Comment (khssupriya) [inline src/app/components/utils/footer/footer.component.html:8]: Let's have GitHub, yoututbe, instagram, linkedin in this order. What do you think?

File: src/app/components/views/my-account/my-account.component.scss [hunk 5]
```diff
@@ -0,0 +1,5 @@
+@import '../../../../styles.scss';
+
+h5{
+    color:#12226b;
```
Comment (khssupriya) [inline src/app/components/views/my-account/my-account.component.scss:4]: Use the colors in the global scss

**Review by khssupriya — CHANGES_REQUESTED** [review]

File: src/app/components/utils/footer/footer.component.scss (hunk 3 above)
Comment (vtejaswini01) [reply src/app/components/utils/footer/footer.component.scss:10]: The footer I pushed is not final.
I just put it to add some data to the footer
I might change the whole structure after myaccount page is done.
Besides, I will make the changes you have mentioned to the final one.

File: src/app/components/utils/footer/footer.component.html (hunk 4 above)
Comment (vtejaswini01) [reply src/app/components/utils/footer/footer.component.html:8]: Yeah, okay

File: src/app/components/views/my-account/my-account.component.scss (hunk 5 above)
Comment (vtejaswini01) [reply src/app/components/views/my-account/my-account.component.scss:4]: ok

**Comment by khssupriya** [comment]
Continued in #14

**Pull request closed without merging**